docs/faq/env_var.md

```diff
@@ -58,6 +58,15 @@ $env:MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
   - Values: Int ```(default=5)```
   - The percentage of GPU memory to reserve for things other than the GPU array, such as kernel launch or cudnn handle space.
   - If you see a strange out-of-memory error from the kernel launch, after multiple iterations, try setting this to a larger value.  
+* MXNET_GPU_MEM_POOL_TYPE
+  - Values: String ```(default=Naive)```
+  - The type of memory pool.
+  - Choices:
+    - Naive: A simple memory pool that allocates memory for the exact requested size and cache memory buffers. If the buffered memory match the requested size, the memory will be returned from the pool in memory allocation.
```

Comment:
If any buffered memory chunk matches the size of a new request, the chunk from the memory pool will be returned and reused.